code/__DEFINES/traits/_helpers.dm

```diff
@@ -0,0 +1,69 @@
+
+// trait accessor defines
+#define ADD_TRAIT(target, trait, source) \
```

Comment:
the \ is to keep it as a macro?

Reply:
It's the newline indicator? It's **required**.